### ISSUE TITLE
Bug in chapter 10 - "npm run service:deploy" includes more than a single stack

### ISSUE THREAD

**Issue by hjoly2003** [stratospheric-dev/stratospheric#222]
### What's going wrong and not working?

Hi,  
when invoking `npm run service:deploy`, the system replies
```bash
> stratospheric-cdk@0.1.0 service:deploy
> cdk deploy --app "./mvnw -e -q compile exec:java -Dexec.mainClass=dev.stratospheric.todoapp.cdk.ServiceApp" --require-approval never
Since this app includes more than a single stack, specify which stacks to use (wildcards are supported) or specify `--all`
Stacks: ServiceParameters-1688911388362 · ServiceStack
```
Indeed, [ServiceApp.java](https://github.com/stratospheric-dev/stratospheric/blob/b6fe2053692519c07eb56adc2344c97d5c49b236/chapters/chapter-10/cdk/src/main/java/dev/stratospheric/todoapp/cdk/ServiceApp.java) defines two stacks: the `parametersStack` and the `serviceStack`.

### Expected outcome

To resolve this issue and have `npm run service:deploy` silently deploying the stacks, you would have to modify the [package.json file](https://github.com/stratospheric-dev/stratospheric/blob/main/chapters/chapter-10/cdk/package.json) so to include an `--all` option to that command:
```json
    "service:deploy": "cdk deploy --all --app \"./mvnw -e -q compile exec:java -Dexec.mainClass=dev.stratospheric.todoapp.cdk.ServiceApp\" --require-approval never",
```

**Comment by rieckpil** [comment]
Thanks for the fix, feel free to provide a PR to fix it in the code :)

**Comment by hjoly2003** [comment]
Doing a follow-up on this issue.

I've found out that when invoking `npm run service:deploy`, two stacks of *ServiceParameters* are created, each with a distinct timestamp in its name. Each has its pair of json files synthetized under the `cdk.out` subdirectory. Somehow, within *ServiceApp.main*, the `long timestamp = System.currentTimeMillis();` snippet is invoked twice! Later, when I run `npm run service:destroy`, it deletes only one of the two *ServiceParameters*.

To resolve the issue on my side, I have removed the timestamp from the name of the *ServiceParameters* stack. Hence, the `service:deploy` command, within the `package.json` file, do not need the `-all` option anymore. Further investigation would be needed as for why the above snippet is executed twice.

Cheers

**Comment by hjoly2003** [comment]
Doing a follow-up on this issue.

I've found out that when invoking `npm run service:deploy` (with the suggested `-all` option), two stacks of *ServiceParameters* are created, each with a distinct timestamp in its name. Each has its pair of json files synthetized under the `cdk.out` subdirectory. Somehow, within *ServiceApp.main*, the `long timestamp = System.currentTimeMillis();` snippet is invoked twice! Later, when I run `npm run service:destroy`, it deletes only one of the two *ServiceParameters*.

To resolve the issue on my side, I have removed the timestamp from the name of the *ServiceParameters* stack. Hence, the `service:deploy` command, within the `package.json` file, do not need the `-all` option anymore. Further investigation would be needed as for why the above snippet is executed twice.

Cheers
